paprika/simulate.py

```diff
@@ -0,0 +1,160 @@
+        self.job['minimization']['cntrl']['ntf'] =              1
+        self.job['minimization']['cntrl']['ntc'] =              1
+        self.job['minimization']['cntrl']['cut'] =              999.0
+        self.job['minimization']['cntrl']['igb'] =              1
+        self.job['minimization']['cntrl']['ntr'] =              None
+        self.job['minimization']['cntrl']['restraint_wt'] =     None
+        self.job['minimization']['cntrl']['restraintmask'] =    None
+        self.job['minimization']['cntrl']['nmropt'] =           1
+        self.job['minimization']['cntrl']['pencut'] =           -1
+        self.job['minimization']['ewald'] =                     None    # or OrderedDict() (Add the other namelists?)
+        self.job['minimization']['wt'] =                        None    # or []
+        self.job['minimization']['DISANG'] =                    'restraints.in'
+        self.job['minimization']['GROUP'] =                     None    # or []
+
+        # Production input file settings
+        self.job['md'] = {}
+        self.job['md']['cntrl'] =                       OrderedDict()
+        self.job['md']['cntrl']['imin'] =               0
+        self.job['md']['cntrl']['ntx'] =                1
+        self.job['md']['cntrl']['irest'] =              0
+        self.job['md']['cntrl']['dt'] =                 0.002
+        self.job['md']['cntrl']['nstlim'] =             5000
+        self.job['md']['cntrl']['ntpr'] =               500
+        self.job['md']['cntrl']['ntwe'] =               0
+        self.job['md']['cntrl']['ntwr'] =               5000
+        self.job['md']['cntrl']['ntxo'] =               1
+        self.job['md']['cntrl']['ntwx'] =               500
+        self.job['md']['cntrl']['ioutfm'] =             1
+        self.job['md']['cntrl']['ntf'] =                2
+        self.job['md']['cntrl']['ntc'] =                2
+        self.job['md']['cntrl']['igb'] =                1
+        self.job['md']['cntrl']['cut'] =                999.0
+        self.job['md']['cntrl']['ntt'] =                3
+        self.job['md']['cntrl']['gamma_ln'] =           1.0
+        self.job['md']['cntrl']['ig'] =                 -1
+        self.job['md']['cntrl']['temp0'] =              298.15
+        self.job['md']['cntrl']['ntr'] =                None
+        self.job['md']['cntrl']['restraint_wt'] =       None
+        self.job['md']['cntrl']['restraintmask'] =      None
+        self.job['md']['cntrl']['nmropt'] =             1
+        self.job['md']['cntrl']['pencut'] =             -1
+        self.job['md']['ewald'] =                       None    # or OrderedDict() (Add the other namelists?)
+        self.job['md']['wt'] =                          None
+        self.job['md']['DISANG'] =                      'restraints.in'
+        self.job['md']['GROUP'] =                       None
+
+        #self.print_time = 1.0 #ps.  Figure out a way to update dicts if this is reset by user. 
```

Comment:
I don't understand your comment. The user should be able to update any of these variables after calling `init`, i.e. 
```
this = gb_simulation()
# gb_simulation.init() is implicitly called now...
this.print_time = 1123412
```
...will override this setting, right?

Reply:
my comment was more a bookmark than decipherable statement.  I was thinking about a way to let the user specify intervals, like printing, in terms of ps rather than timesteps.  So if user set print time to to 5 ps, i need the class to check the dt and then reset the ntpr to however many steps corresponds to 5 ps.  I think this needs @property, but I don't understand them yet.

Reply:
Hmm, not sure I understand why. You just need to know the interval the user specifies -- with units! -- the time step and the number of steps. Then you just do the inverse of this line I have in one of my other scripts.

`print('# NVT Thermalization for {} ns'.format(self.thermalization_steps * self.thermalization_dt / 1000), file=self.log)
`

Reply:
But that basically forces me to use either `self.print_time` approach or the `self.blah.[ntpr]`.  I want init to set the `self.blah.[ntpr]`, but then if the user changes `self.print_time` to something (rather than `None`) then update `self.blah.[ntpr]`.  Not an important feature at the moment ... just something to look into.

Reply:
Okay, so you want initial setting and recalculation?

Reply:
Probably should add this to #18 as well.